tests-unit/execution_test/validate_node_input_test.py

```diff
@@ -0,0 +1,123 @@
+import pytest
+from comfy_execution.validation import validate_node_input
+from enum import StrEnum
```

Comment:
Tests currently run in Python 3.10, which does not have StrEnum support. Can you replace it with raw impl that directly inherit from `str`?